src/Spatial/Euclidean/Circle.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MathNet.Spatial.Euclidean
+{
+    /// <summary>
+    /// Describes a standard 2 dimentional circle
+    /// </summary>
+    public class Circle : IEquatable<Circle>
```

Comment:
We assume namespances here, more consistent with the rest of the lib would be Circle2D.